### ISSUE TITLE
MQTT-less operation

### ISSUE THREAD

**Issue by KortgatGH** [DasLetzteEinhorn/AlphaESS_Monitor_Hass#8]
The "About" states:
" The obtained data that is sent via MQTT to the configured broker _**(or directly to the Home Assistant instance this addon is running on**)_."
Does this mean that one can access the information without MQTT at all? If so, how do I achieve this?

**Comment by DasLetzteEinhorn** [comment]
You're right, this is a bit misleading. You still need a configured MQTT Broker on your HA instance, but you don't need to supply credentials, if it's on the same instance.

**Comment by KortgatGH** [comment]
Thanks very much. I will consider getting up to speed with MQTT
